src/PerfViewCollect/PerfViewCollect.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.5</RuntimeFrameworkVersion>
```

Comment:
I guess I don't care enough to actually figure it out, but this should not have been necessary.   I like to believe that PerfViewCollect is a very generic .NET Core 2.0 application, and as such should not need a specific framework version.

Reply:
Also if an app works before you publish and breaks afterward, it seems likely it is a runtime issue, not an app issue.

Reply:
What was interesting about this is that the actual coreclr.dll that was used in each scenario (shared framework vs. published) was different.

Fortunately (or unfortunately), I can't reproduce this anymore.  Not sure if it was a restore issue or otherwise, but things seem sane now.  Thus, I have removed the RuntimeFrameworkVersion tag.